Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -391,6 +391,34 @@
+    - Botanist
+    - Zookeeper
+
+- type: listing
+  id: UplinkDartGun
+  name: uplink-dart-gun-name
+  description: uplink-dart-gun-desc
+  productEntity: DartGun
+  cost:
+    Telecrystal: 20
+  categories:
+  - UplinkJob
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)